src/mscorlib/shared/System/ReadOnlySpan.cs

```diff
@@ -198,8 +198,18 @@ public bool IsEmpty
         /// </summary>
         public void CopyTo(Span<T> destination)
         {
-            if (!TryCopyTo(destination))
+            // Using "if (!TryCopyTo(...))" results in two branches: one for the length
+            // check, and one for the result of TryCopyTo. Since these checks are equivalent,
+            // we can optimize by performing the check once ourselves then calling Memmove directly.
+
+            if ((uint)_length <= (uint)destination.Length)
```

Comment:
You cannot use nuint for the portable span today. Anything that uses nuint should be under `#if !FEATURE_PORTABLE_SPAN` or `.Fast.cs` files. You may want to copy the shared directory over to corefx and fix it up to make it build - so that it is prepared for mirroring.

Reply:
Copying the shared directly over still builds correctly on corefx, but that's because corefx has its own implementation of `Span` separate from the shared directory implementation. I opened https://github.com/dotnet/corefx/issues/26611 to track making changes to corefx to get it to consume the shared implementation.

In the meantime per our separate conversation we shouldn't let this block this PR.